firmware/thruna/PDM/Src/main.c

```diff
@@ -52,6 +52,8 @@
 #include "configs/App_CurrentLimits.h"
 #include "configs/App_VoltageLimits.h"
 #include "configs/App_HeartbeatMonitorConfig.h"
+
+#include "App_CommitInfo.h"
 /* USER CODE END Includes */
 
 /* Private typedef -----------------------------------------------------------*/
```

Comment:
PDM does not broadcast commit info, despite importing + creating message

Reply:
Fixed! Good catch.